app/models/shopify/webhooks/webhooks_manager.rb

```diff
@@ -0,0 +1,102 @@
+module Shopify
+  module Webhooks
+
+    class WebhooksManager
+      class CreationFailed < StandardError
+      end
+
+      def self.get_manager(params)
```

Comment:
Looks java-ish, what's the benefit compare to juste use `WebhooksManager.new(org_uid, shop_name, token)`?